### PR TITLE
Add docs for color customization on web

### PR DESCRIPTION
**Pull request by mspensieri**
@alavers @Mario54 @lemieux @jugarrit @kvgauthier

### CODE REVIEW

File: source/includes/customization/_styling_conversations_js.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
 ## Styling the Conversation Interface
 
-Using CSS, you can change the style used by Smooch. You can simply inspect any UI elements and override any class in the widget by targeting `#sk-container #sk-wrapper .selector`. Our stylesheet only targets `#sk-container .selector`, it would become less specific than yours.
```
Comment (Mario54) [inline source/includes/customization/_styling_conversations_js.md]: We should keep this text, no?

**Comment by Mario54** [comment]
Can you add a step to iOS to explain that they have to go to the iOS integration page? 
image

**Comment by Mario54** [comment]
Rest is good :+1:

**Comment by lemieux** [comment]
:+1:

**Comment by dannytranlx** [comment]
:+1:

**Comment by alavers** [comment]
:+1:

**Pull request merged**